exotations/solvers/exotica_ddp_solver/src/feasibility_driven_ddp_solver.cpp

```diff
@@ -211,7 +206,7 @@ void FeasibilityDrivenDDPSolver::Solve(Eigen::MatrixXd& solution)
             }
             catch (std::exception& e)
             {
-                // HIGHLIGHT("Error in TryStep: " << e.what())
+                WARNING_NAMED("NaN in ForwardPass", e.what())
```

Comment:
Can the algorithm recover from NaNs here? Shouldn't this abort?

Reply:
It can recover in most cases, but I'd still like to see a warning and know.

Reply:
Then it shouldn't use exceptions. Same reasoning as for FDDP and exceptions in a loop.

Reply:
Technically it is an exception and we do want to terminate the roll-out. It should not happen in a control loop with a well formulated problem but more during problem formulation development.